pages/~/index.js

```diff
@@ -17,6 +18,7 @@ export default function Sub ({ ssrData }) {
   return (
     <Layout sub={variables.sub}>
       <Snl />
+      <NewVisitorBanner />
```

Comment:
Since you named the mutation `hideWelcomeBanner`, I would name this `WelcomeBanner` instead of `NewVisitorBanner`.

Consistency is key in code and life :)